src/components/common/GravatarWithTooltip.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import Gravatar from 'react-gravatar';
+import { Tooltip } from 'reactstrap';
+
+class GravatarWithTooltip extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
```

Comment:
if you write toggle function as an arrow function:
```
toggle = () => {
```

the context will be ok, and you don't need to bind in the constructor.
Personaly I use arrow functions when I write every method in the class that is not React Lifecycle Method.